### PR TITLE
Add info about Builders to all Queries constuctors

### LINKED ISSUE #120: Write about builders in StorIO queries constructors

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #120 

@nikitin-da PTAL

### CODE REVIEW

File: storio/src/main/java/com/pushtorefresh/storio/db/query/DeleteQuery.java [hunk 1]
```diff
@@ -13,6 +13,9 @@
 
     @Nullable public final String[] whereArgs;
 
+    /**
+     * Please use {@link com.pushtorefresh.storio.db.query.DeleteQuery.Builder} instead of constructor
+     */
     public DeleteQuery(@NonNull String table, @Nullable String where, @Nullable String[] whereArgs) {
```
Comment (nikitin-da) [inline storio/src/main/java/com/pushtorefresh/storio/db/query/DeleteQuery.java]: Do we really need public constructor?

Comment (artem-zinnatullin) [inline storio/src/main/java/com/pushtorefresh/storio/db/query/DeleteQuery.java]: See #119, what do you think?

Comment (nikitin-da) [inline storio/src/main/java/com/pushtorefresh/storio/db/query/DeleteQuery.java]: What about protected?

Comment (artem-zinnatullin) [inline storio/src/main/java/com/pushtorefresh/storio/db/query/DeleteQuery.java]: Yep, it should work, can you please test it? I am working on Sample app..

Comment (nikitin-da) [inline storio/src/main/java/com/pushtorefresh/storio/db/query/DeleteQuery.java]: No problem

**Pull request merged**